elastic/kibana
# [SecuritySolution] Migrate ESS get started page to ess_security plugin

## Summary
Original issue: https://github.com/elastic/security-team/issues/6489
Relevant issue: https://github.com/elastic/kibana/pull/158461 | https://github.com/elastic/kibana/pull/158955

1. Move Landing Cards Component to ess_security plugin
2. Any component render `LandingPageComponent` when no indices exists will be redirected to get started page. 

**Steps to verify:**
1. Start an empty Elastic Search to verify the get started page, then visit all the pages that render get started page when no indices exist to see if they are rendered properly.
2. Add some data to Elastic Search and visit each page and see if it's rendered properly. Open the timeline, and flyout, make sure they are looking alright on the page.

ESS `yarn start`:

Get started page:
No data view exists:
<img width="2549" alt="ess-get-started" src="https://github.com/elastic/kibana/assets/6295984/dd560ab3-e3b1-4695-adfa-a1851da04cb0">




Serverless `yarn serverless-security`:

Get started page:
<img width="2553" alt="serverless-get-started" src="https://github.com/elastic/kibana/assets/6295984/2839c66f-e675-4eb5-b909-1b7af450b6d1">

No data view exists:
<img width="2542" alt="serverless-no-data" src="https://github.com/elastic/kibana/assets/6295984/d8e3a26b-3d39-442a-98ca-df0831e332ea">

Data exists:

<img width="2539" alt="Screenshot 2023-06-14 at 23 30 34" src="https://github.com/elastic/kibana/assets/6295984/ab7b413e-f25a-4c5e-9946-f4094a26fcdb">





### Checklist

Delete any items that are not applicable to this PR.
